shared/specs/notifications-openapi.yaml

```diff
@@ -92,8 +144,7 @@ components:
           type: string
           format: date-time
         read:
-          type: string
-          format: date-time
+          type: number
```

Comment:
why this change? We now receive millisecond timestamp instead of formatted string?

Reply:
we had just 'create'. the field was always null. now we are retrieving this field and I realized it is a number

Reply:
we dont care about the value. just the fact that it was read